public/app/shared/utilService.js

```diff
@@ -89,29 +89,6 @@
     };
 
     /**
-     * URL of Dashboard tool, if added to the course
-     *
-     * @param  {Object}       course          Course with one or more SuiteC tools
-     * @return {String}                       URL without an appended user id
-     */
-    var getGenericDashboardUrl = function(course) {
-      return course.dashboard_url;
-    };
-
-    /**
-     * If the appropriate LTI tool has been added to the course site then
-     * return URL of profile per user id.
-     *
-     * @param  {Object}       course          Course with one or more SuiteC tools
-     * @param  {Object}       user            Member of course
-     * @return {String}                       URL with necessary query args
-     */
-    var getUserDashboardUrl = function(course, user) {
-      var dashboardUrl = getGenericDashboardUrl(course);
-      return dashboardUrl && user ? dashboardUrl + '?_id=' + user.id : null;
-    };
-
-    /**
```

Comment:
@paulkerschen The above logic was moved to a directive. Appropriate? Better? I'll let you be the judge.